### PR TITLE
Add picture to docs

### PR DESCRIPTION
**Pull request by yngvar-antonsson**
Follow-up for #1843

### CODE REVIEW

File: rst/topics/failover.rst [hunk 1]
```diff
@@ -274,7 +274,19 @@ In Lua API:
 
 You could also make node unelectable in WebUI:
```
Comment (xuniq) [inline rst/topics/failover.rst:275]:
```suggestion
You can also make a node unelectable in WebUI:
```
Please don't substitute **could** for **can** unless you're referring to the past:
1. In previous versions of the program, you could create a project in one click. - okay
2. Should you need it, you could ask for help in our Telegram chat. - not okay!

See [this page](https://www.notion.so/tarantool/Modal-verbs-in-technical-documentation-1f2b82c4f25d4e53bd3564b39c45e438) in Notion for details.

File: rst/topics/failover.rst [hunk 2]
```diff
@@ -274,7 +274,19 @@ In Lua API:
...
 
-**ADD PICTURE**
+..  image:: images/set_non_electable.png
+    :align: left
+    :scale: 40%
+
+|nbsp|
+
+If everything is ok, you will see crossed out crown to the left of instance name.
```
Comment (xuniq) [inline rst/topics/failover.rst:283]:
```suggestion
If everything is ok, you will see a crossed-out crown to the left of the instance name.
```

File: rst/topics/failover.rst (hunk 1 above)
Comment (xuniq) [inline rst/topics/failover.rst:275]: I also suggest the following change:

> You can restrict node to be elected in ``stateful`` failover mode by GraphQL
> or Lua API. "Unelectable" node can't become a leader in replicaset.

You can restrict the election of a particular node in the ``stateful``failover mode by GraphQL or Lua API. An "unelectable" node can't become a leader in a replicaset.

**Comment by yngvar-antonsson** [comment]
@xuniq please help with the images in PR. Do I use correct path for them?
[Here](https://github.com/tarantool/cartridge/blob/943c7b4b0f7cda08b7bc186af800eeeaac7b33fc/rst/topics/failover.rst) I see that something's wrong.

**Pull request merged**